src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+  const married = isMarried
+    ? `${partnerName} is my ${partner}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age > 0 && <p className="Person__age">I am {age}</p>}
```

Comment:
The condition `age > 0` assumes that `age` is always defined. Consider using a more robust check, such as `age !== undefined && age > 0`, to handle cases where `age` might be missing.